exasol_script_languages_developer_sandbox/runtime/ansible/roles/jupyter/files/notebook/script-languages.ipynb

```diff
@@ -0,0 +1,1135 @@
+{
```

Comment:
Storing notebook files can get tedious and error prone especially when it comes to merging when you store  outputs.
You may wan't to have a look at this article: [The Jupyter+git problem is now solved](https://www.fast.ai/posts/2022-08-25-jupyter-git.html)